server/config/nl_page/topic_cache.json

```diff
@@ -15422,8 +15614,10 @@
       ],
       "relevantVariableList": [
         "UnemploymentRate_Person",
-        "UnemploymentRate_Person_Female",
-        "UnemploymentRate_Person_Male"
+        "dc/svpg/UnemploymentRateByGender",
+        "UnemploymentRate_Person_Male",
```

Comment:
Added a note to remove UnemploymentRate_Person_Male as it is within dc/svpg/UnemploymentByGender